services/token-reward-service/.env

```
@@ -8,9 +8,9 @@ DB_PORT=5433
 DB_NAME=course_db
 
 WEB3_PROVIDER_URL=http://127.0.0.1:7545
-ACCOUNT_PRIVATE_KEY=0xaba9dede35ae3042e09949a82605a33f77b1602a6cc463ee7b829d663accf570
+ACCOUNT_PRIVATE_KEY=0x22821a351140957d1c69a151ac67d691c8f2671991d1479ac80bdfa5723bf291
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ACCOUNT_PRIVATE_KEY key should go before the WEB3_PROVIDER_URL key

(UnorderedKey)